utils/lib/Quickstart.ts

```diff
@@ -52,8 +52,8 @@ const ConfigToMutation: ConfigToMutationMap[] = [
   { configKey: 'alertPolicies', mutationKey: 'alertConditions', ctor: Alert },
   { configKey: 'dashboards', mutationKey: 'dashboards', ctor: Dashboard },
   {
-    configKey: 'dataSourceIds',
-    mutationKey: 'dataSourceIds',
+    configKey: 'dataSources',
+    mutationKey: 'dataSources',
```

Comment:
```suggestion
    mutationKey: 'dataSourceIds',
```

Reply:
We will still be mapping to `dataSourceIds` when submitting the mutation to nerdgraph